gradle.properties

```diff
@@ -0,0 +1,2 @@
+android.enableJetifier=true
```

Comment:
Do we require jetifier ? We should try to remove the old appcompat dependencies as much as possible.